snmp/assets/monitors/high_cpu.json

```diff
@@ -0,0 +1,31 @@
+{
+	"version": 2,
+	"created_at": "2023-12-26",
+	"last_updated_at": "2023-12-26",
+	"title": "SNMP device CPU usage is high",
+	"description": "CPU usage measures the percentage of processing capacity that is in use and provides insights into the overall health and efficiency of the device. This monitor alerts if the CPU usage goes above 90%, suggesting potential performance issues",
+	"definition": {
+		"name": "[SNMP] CPU usage high for {{snmp_device.name}} in namespace {{device_namespace.name}}",
+		"type": "query alert",
+		"query": "avg(last_5m):avg:snmp.cpu.usage{*} by {snmp_device,device_namespace} >= 90",
+		"message": "{{#is_alert}} \n{{snmp_device.name}} in namespace {{device_namespace.name}} is reporting high CPU usage (at or above 90%).\n{{/is_alert}} \n\n{{#is_recovery}} \nCPU usage for {{snmp_device.name}} in namespace {{device_namespace.name}} is back to normal.\n{{/is_recovery}}",
```

Comment:
You may want to add a description for `{{#is_warning}}`, otherwise there is no content on the warning message.
(similar comment for similar monitors)